financial-templates-lib/logger/Logger.js

```diff
@@ -62,7 +62,7 @@ const Logger = winston.createLogger({
     winston.format.json()
   ),
   transports,
-  exitOnError: false
+  exitOnError: process.env.EXIT_ON_ERROR ? process.env.EXIT_ON_ERROR : false
```

Comment:
The default behavior for this should be set to false. however, sometimes you might want it to be true (like in the serverless context).

Reply:
This is because the `ENVIROMENT` variable is already used in other places to inform log specific information, like in the `Transports.js` where `process.env.ENVIRONMENT == "production"` is used to inform if the stack driver logging should be added.

I could set this that it would only set `exitOnError` IFF the `ENVIROMENT==serverless` but I wanted to be able to set these things independently as I'm still not 100% sure we want to even use this `exitOnError` in the serverless context.